Weirdness in TCPcloseSocket

Have a mosey at this
```
CS_TCP.c: In function ‘TCPcloseSocket’:
CS_TCP.c:203:1: warning: control reaches end of non-void function [-Wreturn-type]
 }  // end of TCPcloseSocket
```

--- a/CS_TCP.c
+++ b/CS_TCP.c
@@ -199,6 +199,7 @@ int TCPcloseSocket (SOCKET socket2close)
     {
         socketCount--;  // decrement the socket counter
         printf("Socket closed, %d remaining\n", socketCount);
+		return EXIT_SUCCESS;
     }
 }  // end of TCPcloseSocket
 

